app/helpers/json_ld_helper.rb

```diff
@@ -134,7 +134,7 @@ def patch_for_forwarding!(original, compacted)
         patch_for_forwarding!(value, compacted_value)
       elsif value.is_a?(Array)
         compacted_value = [compacted_value] unless compacted_value.is_a?(Array)
-        return if value.size != compacted_value.size
+        return nil if value.size != compacted_value.size
```

Comment:
Do I understand correctly that returning `nil` is enough not to trip the cop? That feels a bit surprising.